generator/template.go

```diff
@@ -58,7 +58,7 @@ export type {{.Name}} = {
   }
 {{- else }}
   static {{.Name}}(req: {{tsType .Input}}, initReq?: fm.InitReq): Promise<{{tsType .Output}}> {
-    return fm.fetchReq<{{tsType .Input}}, {{tsType .Output}}>(` + "`{{renderURL .}}`" + `, {...initReq, {{buildInitReq .}}})
+    return fm.fetchReq<{{tsType .Input}}, {{tsType .Output}}>(` + "`{{renderURL .}}`" + `,  {{buildURLQueryParams .}} {...initReq, {{buildInitReq .}}})
```

Comment:
`buildURLQueryParams` in compile time doesn't have any knowledge on how to handle runtime payload properly as discussed in #9  I would say

Reply:
Yea, and so I have been thinking about how to implement this and the only way seems to be:
* Add a TypeScript function within the template which builds the URL Query parameters during runtime by analyzing the request payload and URL Path.

Let me know what you think about this idea 🙏🏼

Reply:
a runtime function should do the work.

Reply:
Thank you for your feedback! I do have an idea about how to implement the runtime function and will update this PR soon 🙇🏼